pkgs/applications/graphics/tikzit/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, qmake, qt5, bison, flex }:
+
+stdenv.mkDerivation rec {
+  name = "tikzit-${version}";
+  version = "2.0";
+
+  src = fetchurl {
+    url = "https://github.com/tikzit/tikzit/archive/v${version}.tar.gz";
+    sha256 = "15gjnkm4nzm49bmmsfs4xs24vknr7i16fwi1830rh8mcp0qlba6i";
+  };
+
+  nativeBuildInputs = [ qmake bison flex ];
+
+  meta = with stdenv.lib; {
+    description = "TikZiT is a graphical tool for rapidly creating graphs and string diagrams using PGF/TikZ.";
```

Comment:
the period at the end should be removed according to https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md